lib/Analysis/SelectVariableNames/SelectVariableNames.cpp

```diff
@@ -0,0 +1,40 @@
+#include "include/Analysis/SelectVariableNames/SelectVariableNames.h"
+
+#include <iostream>
```

Comment:
nit: i always keep this in after debugging by accident, do you need this?

Reply:
Done, thanks!